infrastructure/data/README.md

```diff
@@ -118,7 +118,7 @@ Note that when we apply this change, Pulumi will first spin up a new db, with a
   
     c. We are going to replace this value with the URI or 'endpoint' of the new db. This should be as simple as swapping the current `instance` part of the `db-host` string for the new db name (`app...`) which we grabbed in step 5, but it's worth verifying this by clicking through into the new db in the AWS console, and comparing your construction against the `Endpoint` value given there (literally do `Ctrl + F`).
 
-    d. Finally, run `pulumi config set --secret db-host [new-endpoint]`, commit the change, push it up, and get the PR approved and ready to merge (but don't merge it yet - we'll do that in step 7). Recall that unlike the data layer, the application layer is deployed automatically via CI/GitHub action ([staging](https://github.com/theopensystemslab/planx-new/actions/workflows/push-main.yml) / [prod](https://github.com/theopensystemslab/planx-new/actions/workflows/push-production.yml)).
+    d. Finally, run `esc env set application/{stack} pulumiConfig.application:db-host [new-endpoint] --secret db-host`, commit the change, push it up, and get the PR approved and ready to merge (but don't merge it yet - we'll do that in step 7). Recall that unlike the data layer, the application layer is deployed automatically via CI/GitHub action ([staging](https://github.com/theopensystemslab/planx-new/actions/workflows/push-main.yml) / [prod](https://github.com/theopensystemslab/planx-new/actions/workflows/push-production.yml)).
```

Comment:
based on your edit in the how-to on adding a secret, you shouldn't need the `db-host` at end of this replaced command?

Reply:
Another typo - that's correct thank you! 🙌